airbyte-db/lib/src/main/java/io/airbyte/db/SqlDatabase.java

```diff
@@ -27,29 +27,10 @@
 import com.fasterxml.jackson.databind.JsonNode;
 import java.util.stream.Stream;
 
-public abstract class SqlDatabase implements AutoCloseable {
-
-  private JsonNode sourceConfig;
-  private JsonNode databaseConfig;
+public abstract class SqlDatabase extends NoSqlDatabase {
```

Comment:
Looks like we have a naming conflict here. `SqlDatabase` should not be an extension of `NoSqlDatabase`. 
As NoSqlDatabase doesn't contain any `no sql` logic, I propose to rename `NoSqlDatabase`.
For example, `Database`, `AbstractDatabase` or something like that.